guides/common/modules/con_managing-dhcp-using-smart-proxies.adoc

```diff
@@ -0,0 +1,16 @@
+[id="Managing_DHCP_Using_Smart_Proxies_{context}"]
+= Managing DHCP Using {SmartProxies}
+
+{SmartProxies} can manage DHCP within their network.
+You can use the DHCP modules for {SmartProxy} to query for available IP addresses, add new, and delete existing reservations.
+Note that your {SmartProxy} cannot manage subnet declarations.
+ifdef::foreman-el,foreman-deb,katello[]
+For more information on managing subnet declarations, see https://github.com/theforeman/puppet-dhcp[puppet-dhcp].
+endif::[]
```

Comment:
This ifdef block really belongs in a part of the manual, if any at all. Mostly because we don't have instructions about how to map the DHCP options in that module to what you can use in the installer. Right now I'm leaning to dropping it

Reply:
This point is still open.

Reply:
I think so yes.

Reply:
Removed.